toolkit/scripts/srpm_pack.mk

```diff
@@ -19,6 +19,14 @@ SRPM_BUILD_LOGS_DIR = $(LOGS_DIR)/pkggen/srpms
 toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
 srpm_pack_list_file = $(BUILD_SRPMS_DIR)/pack_list.txt
 
+ifneq ($(strip $(SRPM_PACK_LIST)),)
```

Comment:
I'm curious: is there a way to re-assign `SRPM_PACK_LIST` with its stripped version, so we don't have to repeat this everywhere? Not a blocker for this PR, this wasn't introduced by you here anyway, but if that is possible we might as well clean-up a little.